doc/man3/BN_cmp.pod

```diff
@@ -35,11 +42,16 @@ of I<a> and I<b>.
 BN_is_zero(), BN_is_one() BN_is_word(), BN_abs_is_word() and
 BN_is_odd() return 1 if the condition is true, 0 otherwise.
 
+BN_are_coprime() returns 1 if the B<BIGNUM>'s are coprime, otherwise it
+returns 0.
+
 =head1 HISTORY
 
 Prior to OpenSSL 1.1.0, BN_is_zero(), BN_is_one(), BN_is_word(),
 BN_abs_is_word() and BN_is_odd() were macros.
 
+The function BN_are_coprime() was added in OpenSSL 3.2.
```

Comment:
The docs in this say BN_are_coprime() was added in 3.2, but this is tagged for 3.1.

Reply:
Well spotted.. I definitely want this in 3.1 (It was the main reason for the change).